helm/adelphi/templates/nosqlbench-jobs.yaml

```diff
@@ -12,6 +12,7 @@ spec:
       image: nosqlbench/nosqlbench:latest
       args: [
         "--show-stacktraces",
+        "--report-csv-to=/results/nosqlbench-source",
```

Comment:
"nosqlbench-source" is... actually the CSV file itself and not a directory?  I ask only because the Gemini template explicitly did a "mkdir" before running the app.  If this is intended to be the CSV file itself then clearly making a directory is unnecessary... just wanted to understand.

Reply:
NoSQLBench writes a bunch of output files into that folder.